addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -2573,6 +2573,49 @@ var SnippetsMenu = Widget.extend({
             }
         });
 
+        //avoid dropzone collisions
+
+        function adjustMargin($zone, marginType, overlap, minMargin, maxMargin) {
+            let currentMargin = parseFloat($zone.css(marginType));
+            let newMargin = Math.min(Math.max(currentMargin + overlap, minMargin), maxMargin);
+            $zone.css(marginType, newMargin + "px");
```

Comment:
- We are removing jQuery in master, when you add new code let's not use jQuery (don't modify existing code, just make sure that new code does not use jQuery)
- Nitpicking, try to use template literals for this:
```javascript
`${newMargin}px`
```